### ISSUE TITLE
Change Safe validation schema to allow null Safe version

### ISSUE THREAD

**Issue by hectorgomezv** [safe-global/safe-client-gateway-nest#568]
**Problem**
Some validation errors are being caused by Safes with a `null` `version` field.
Example [here](https://safe-transaction-gnosis-chain.safe.global/api/v1/safes/0xa08826D91a104d31856F448190B346f077e3EABb/).
More context [here](https://github.com/safe-global/safe-transaction-service/issues/1571).

**Solution**
Once the Safe{Core} team confirms this is a rare but expected case, make `version` field `nullable` in the Safe validation schema. Adjust OpenAPI schema and/or tests if necessary.

**Comment by fmrsabino** [comment]
I don't think this is a bug. We can consider that these Safes are invalid also. One of the goals of the Safe Client Gateway is to provide to the UI clients a formatted response that we support. Right now we support safes that have a valid version.
